docs/config/README.md

```diff
@@ -1414,7 +1414,7 @@ disabled = false
 
 ## Java
 
-The `java` module shows the currently installed version of Java.
+The `java` module shows the currently installed version of [Java](https://www.java.com/).
```

Comment:
That website still links to an I think non-updated Java 8 download.
Wikipedia has this as the official site:
```suggestion
The `java` module shows the currently installed version of [Java](https://www.oracle.com/java/).
```

Reply:
That's quite surprising... TIL.